Source/Core/GoogleEarthEnterpriseTileInformation.js

```diff
@@ -39,7 +39,7 @@ define([
     /**
      * Creates GoogleEarthEnterpriseTileInformation from an object
      *
-     * @param {Object} info Object to be cloned
+     * @param {*} info Object to be cloned
```

Comment:
This one sounds like it still should be of type `Object`. If not, change the doc wording to be more general to all types.